### ISSUE TITLE
WebSocketApp proxy auth settings aren't properly documented

### ISSUE THREAD

**Issue by hazeled** [websocket-client/websocket-client#809]

**Comment by engn33r** [comment]
Adding additional information. This is a blank section in the documentation described as "coming soon" and should be filled in later with example code.
https://websocket-client.readthedocs.io/en/latest/examples.html#connecting-through-a-proxy

**Comment by hazeled** [comment]
It seems to work correctly using the following code:

```python
ws.run_forever(proxy_type = 'socks5', http_proxy_host = proxy_ip, http_proxy_auth = (proxy_username, proxy_password,))
```

There is one issue - When the WebSocketApp is closed, no message or code is passed to the code function. Kinda makes things hard to debug 😅

**Comment by engn33r** [comment]
The close message or code is a separate topic but #708 sounds similar to what you are asking.

**Comment by engn33r** [comment]
Thanks for sharing your example code. Fixed in commit bd84b50
